php/commands/cron.php

```diff
@@ -385,7 +385,7 @@ protected static function get_cron_events() {
 			foreach ( $hooks as $hook => $hook_events ) {
 				foreach ( $hook_events as $sig => $data ) {
 
-					$events["$hook-$sig"] = (object) array(
+					$events["$hook-$sig-$time"] = (object) array(
```

Comment:
Is this key actually used for anything, or could we always just add to the array?